### DISCUSSION TITLE
Do I have to upgrade the version of Jedis？

### DISCUSSION THREAD

**Discussion by Jase-kk** [xetorthio/jedis#4027] [Q&A] [answered]
I'm using redis 5.0.13 and jedis 2.9.3
It doesn't seem to be a problem，because they work well together
But，I see the jedis verion for redis 5.X is 3.9.X or 4.X
I should upgrade or keep unchanged？

**Comment by sazzad16** [comment, marked as answer]
It would be possible to use Jedis 5.x for Redis 5.x. It's just that we're not _officially_ testing it anymore.

Only issue I can think of is STRALGO command which is removed in Jedis 5. If you're not using that command, there should be no issue.

**Reply by Jase-kk** [reply]
Thank you for answer, it really helps me
